Xamarin.Forms.Platform.Android/AppCompat/FrameRenderer.cs

```diff
@@ -286,5 +290,22 @@ void UpdateShadow()
 			else
 				CardElevation = 0f;
 		}
+
+		void UpdateCornerRadius()
+		{
+			if (_defaultCornerRadius == -1f)
+			{
+				_defaultCornerRadius = Radius;
+			}
+
+			float cornerRadius = Element.CornerRadius;
+
+			if (cornerRadius == -1f)
```

Comment:
I was thinking negative values do not make sense anyway. Beside -1, if someone uses it, then unexpected behavior can happen.